src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -754,7 +754,11 @@ internal static bool IsSelfSigned(X509Chain chain)
 
         internal static AsymmetricAlgorithm GetAnyPublicKey(X509Certificate2 certificate)
         {
-            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey() ?? certificate.GetDSAPublicKey();
+            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey()
+#if netcoreapp || uap || uapaot
```

Comment:
@bartonjs so if I understand correctly we would need to port every new API to netfx? If yes I'd rather have only netcoreapp+uap